lib/shopify-cli/messages/messages.rb

```diff
@@ -48,14 +48,28 @@ module Messages
         config: {
           help: <<~HELP,
           Change configuration of how the CLI operates
-            Usage: {{command:%s config [ feature ] [ feature_name ] }}
+            Usage: {{command:%s config [ feature | analytics ] }}
           HELP
           feature: {
+            help: <<~HELP,
+            Change configuration of various features
+              Usage: {{command:%s config [ feature ] [ feature_name ] }}
+            HELP
             enabled: "{{v}} feature {{green:%s}} was enabled",
             disabled: "{{v}} feature {{green:%s}} was disabled",
             is_enabled: "{{v}} feature {{green:%s}} is enabled",
             is_disabled: "{{v}} feature {{green:%s}} is disabled",
           },
+          analytics: {
+            help: <<~HELP,
+            Opt in/out of anonymous usage reporting
+              Usage: {{command:%s config [ analytics ] }}
+            HELP
+            enabled: "{{v}} analytics was enabled",
+            disabled: "{{v}} analytics was disabled",
+            is_enabled: "{{v}} analytics is enabled",
+            is_disabled: "{{v}} analytics is disabled",
```

Comment:
I can see this is an existing pattern, but the use of "was" for something that just happened seems a little surprising to me. Thoughts on switching this to "has been enabled" or "is now enabled"?

🤔 Also, in my head, "analytics" are a plural (`were`/`are`), not a singular thing, but maybe that's just me? Do we have a rule for this, @gfscott?

Reply:
Ah, nice catch! Our rule is to default to standard rules of grammar unless there's a good reason not to. So "analytics were disabled" should be the way to go. And I agree we can give a stronger distinction than just `was`/`is`:

```suggestion
            enabled: "{{v}} feature {{green:%s}} have been enabled",
            disabled: "{{v}} feature {{green:%s}} have been disabled",
            is_enabled: "{{v}} feature {{green:%s}} are currently enabled",
            is_disabled: "{{v}} feature {{green:%s}} are currently disabled",
          },
          analytics: {
            help: <<~HELP,
            Opt in/out of anonymous usage reporting
              Usage: {{command:%s config [ analytics ] }}
            HELP
            enabled: "{{v}} analytics have been enabled",
            disabled: "{{v}} analytics have been disabled",
            is_enabled: "{{v}} analytics are currently enabled",
            is_disabled: "{{v}} analytics are currently disabled",
```

Reply:
Grammar is NOT my strong suit, but should the messages for the feature command be:
`{{v}} feature {{green:%s}} **has** been enabled`
`{{v}} feature {{green:%s}} **is** currently enabled`
?

Reply:
`feature debug is current enabled` <> `feature debug are currently disabled` 
it seems awkward.

Reply:
Yeah, to be clear this should only be for the `analytics` case, not `feature debug` — I think I just misread the top four lines and thought they were also related to analytics.

I'd say for `debug`, however, we should also switch to the style "has been disabled" and "is currently disabled" etc.